cmd/neofs-lens/internal/meta/list.go

```diff
@@ -40,15 +39,12 @@ func listFunc(cmd *cobra.Command, _ []string) error {
 		return fmt.Errorf("%s flag must be positive", limitFlagName)
 	}
 
-	var prm meta.ListPrm
-	prm.SetCount(vLimit)
-
-	res, err := db.ListWithCursor(prm)
+	addrs, _, err := db.ListWithCursor(int(vLimit), nil)
```

Comment:
why int?

Reply:
Because that's the type `ListPrm` had for `count`.